src/modules/roc_audio/depacketizer.cpp

```diff
@@ -208,5 +216,19 @@ packet::PacketPtr Depacketizer::read_packet_() {
     return pp;
 }
 
+void Depacketizer::set_frame_flags_(Frame& frame,
+                                    const size_t dropped_packets,
+                                    const packet::timestamp_t missing_samples) {
+    if (dropped_packets != dropped_packets_) {
+        frame.add_flags(Frame::FlagSkip);
+    }
+
+    const size_t diff = (size_t)ROC_UNSIGNED_SUB(packet::signed_timestamp_t,
+                                                 missing_samples_, missing_samples);
```

Comment:
We don't need ROC_UNSIGNED_SUB here, because we always cast the result to the unsigned type. It was also unnecessary in some other places, I've removed them: 71d93382df1e916e06dd2b1aa503c3d6ae8cae63

Reply:
Fixed